### DISCUSSION TITLE
请问美国通过法案在未来5年投资15亿元专门撰写有关中国的负面报道是否属于言论自由

### DISCUSSION THREAD

**Discussion by xyzhou-1** [The-Run-Philosophy-Organization/run#394] [General]
连个中国脱贫纪录片都被全美下架,美国人,你要自信.反正我们输麻了,怕什么.

想从发展中国家跑到发达国家无可厚非,是提升物质水平最快捷的方式.掰扯那么多就有点搞笑了.

**Comment by the-run-philosophy** [comment]
我觉得这笔钱被贪污了，需要15亿？给我1亿美金我帮他全部搞定。
还需要什么专门撰写啊？？
把微信上微博上被删除的上海疫情录音、录像、亲身经历全部一股脑直接登上经济学人，华尔街日报，纽约时报呢和美国新闻联播头版头条不就好了？？？？？？？？
什么，美国没有新闻联播？？？？？

**Reply by xyzhou-1** [reply]
什么辩论逻辑。顾左右而言其他。这个法案又不是我自己编的。
我问的是美国通过金钱控制舆论是否属于你说的言论自由。结果句句不离中国。就这审题能力，还扯哲学呢。
美国不会删号删评论？我信了。
“连个中国脱贫纪录片都被全美下架”，这个也没有回答我。

什么，美国棱镜门是假的？？？维基解密是假的？？？
